kolena/_experimental/object_detection/workflow.py

```diff
@@ -75,6 +76,7 @@ class TestSampleMetricsSingleClass(MetricsTestSample):
     has_TP: bool
     has_FP: bool
     has_FN: bool
+    ignored: bool
```

Comment:
Thoughts on naming? The other bools are `has_*`, but I didn't want to make this `is_ignored` as it's still different, but more verbose.